### DISCUSSION TITLE
Threshold config settings don't seem to work as it should

### DISCUSSION THREAD

**Discussion by studioab** [lost-pixel/lost-pixel#264] [General]
Hi,

here is my config
![image](https://user-images.githubusercontent.com/13108111/236913969-71155689-6f87-4261-8d08-481981bda55c.png)

according to documentation, these should fail

<img width="788" alt="image" src="https://user-images.githubusercontent.com/13108111/236913542-559d3206-02f8-4cc7-a3a6-d913163eb705.png">

**Comment by d-ivashchuk** [comment]
Hey! Do you mind sharing the version of lost pixel that you are running? From top level the setup looks good 

Just an idea - are you setting thresholds on the level of individual story? That has higher specificity and overrides the general config

**Reply by studioab** [reply]
this is global file hosted in the root
![image](https://github.com/lost-pixel/lost-pixel/assets/13108111/b1e05239-9a25-4a2e-be9c-d4a90ad64e20)

![image](https://github.com/lost-pixel/lost-pixel/assets/13108111/d6cb1f42-473b-4376-b768-6a0e9aabfa88)

**Comment by chriskalmar** [comment]
Hey @studioab 
this seems fine.

From our [docs](https://docs.lost-pixel.com/user-docs/api-reference/lost-pixel.config.js-or-ts#options):
>Values between 0 and 1 are interpreted as percentage of the image size.
>Values greater or equal to 1 are interpreted as absolute pixel count.

This means `0.01` in your config stands for `1%`.
The logs display percentages. 
We might need to provide some examples in our docs to make it more obvious.

So, if you would like the threshold to be at `0.01%` you would need to set it to `0.0001`.
I hope this helps.

**Reply by studioab** [reply]
Makes sense now :) having it in the docs would be helpful. Thanks

**Reply by chriskalmar** [reply]
Awesome. We will add more details to our docs.
I'll close this now. 
In case it should not work, please just reopen the issue.

**Comment by chriskalmar** [comment]
We've updated our docs.
Please check it out @studioab 
https://docs.lost-pixel.com/user-docs/recipes/general-recipes/thresholds

**Reply by studioab** [reply]
Love it. Great job explaining it.
